src/templates/tpl-submissions.html

```diff
@@ -141,7 +141,7 @@
 				<label for="lifeStatus" class="afch-label">Is the subject alive?</label>
 				<select id="lifeStatus" class="afch-input">
 					<option value="unknown" selected>Unknown</option>
-					<option value="living">Living</option>
+					<option value="blp">Living</option>
```

Comment:
Internally in AFCH (but not in the wikicode), can we keep using "living" as the value? I find living to be more readable.